nsv13/code/modules/overmap/overmap.dm

```diff
@@ -18,6 +18,7 @@
 	animate_movement = NO_STEPS // Override the inbuilt movement engine to avoid bouncing
 	req_one_access = list(ACCESS_HEADS, ACCESS_MUNITIONS, ACCESS_SEC_DOORS, ACCESS_ENGINE) //Bridge officers/heads, munitions techs / fighter pilots, security officers, engineering personnel all have access.
 
+	move_resist = INFINITY //THIS MAY BE A BAD IDEA - FIND TAG: BURN ME, NO!!!!!
```

Comment:
Alarming, but sure

Reply:
I haven't actually seen what var this needs to be to block telekinesis, nor have I tested it at this point

Reply:
This will block pulls IIRC. Look at the mecha update PR to see what to do